packages/shared/lib/models/NangoConfig.ts

```diff
@@ -118,15 +120,22 @@ export interface NangoSyncConfig {
     returns: string[];
     models: NangoSyncModel[];
     endpoints: NangoSyncEndpoint[];
+    is_public?: boolean;
+    pre_built?: boolean;
```

Comment:
what's the difference between is_public and pre_built?

Reply:
Pre built is what we deliver to the customer and public is the integrations in the https://github.com/NangoHQ/nango/tree/master/integration-templates